src/main/java/core/selenium/WebElementAction.java

```diff
@@ -29,23 +29,11 @@ public void waitForVisible(final WebElement webElement) {
         WebDriverManager.getInstance().getWait().until(ExpectedConditions.visibilityOf(webElement));
     }
 
-    /**
-     * Sets a webElement with a value.
-     *
-     * @param webElement that will be set.
-     * @param typeText   value that will set.
-     */
-    public void setInputField(final WebElement webElement, final String typeText) {
-        waitForVisible(webElement);
-        webElement.clear();
-        webElement.sendKeys(typeText);
-    }
-
     /**
      * Sends keys to the given WebElement.
      *
      * @param selector represents a selector
-     * @param input is the data to introduce
+     * @param input    is the data to introduce
      */
     public void setInputField(final By selector, final String input) {
         WebDriverManager.getInstance().getWebDriver().findElement(selector).clear();
```

Comment:
Since the driver is called many times, it could be set as an attribute so we don't repeat this sentence in the methods and just call the "driver" attribute.